app/export.go

```diff
@@ -135,8 +149,8 @@ func (app *App) prepForZeroHeightGenesis(ctx sdk.Context, jailWhiteList []string
 
 	// Iterate through validators by power descending, reset bond heights, and
 	// update bond intra-tx counters.
-	store := ctx.KVStore(app.keys[staking.StoreKey])
-	iter := sdk.KVStoreReversePrefixIterator(store, staking.ValidatorsKey)
+	store := ctx.KVStore(app.keys[stakingtypes.StoreKey])
+	iter := sdk.KVStoreReversePrefixIterator(store, stakingtypes.ValidatorsKey)
 	counter := int16(0)
 
 	for ; iter.Valid(); iter.Next() {
```

Comment:
The next line `addr := sdk.ValAddress(iter.Key()[1:])` should be changed to `		addr := sdk.ValAddress(stakingtypes.AddressFromValidatorsKey(iter.Key()))`.

The keys are now `prefix + addrLength + addressbytes`, and the helper `AddressFromValidatorsKey` accounts for this by using `key[2:]` instead of `key[1:]`